lib/routes/geekpark/index.ts

```diff
@@ -0,0 +1,219 @@
+            category: [...new Set([...item.tags, item.column?.title])].filter(Boolean),
+            author: item.authors.map((a) => a.realname ?? a.nickname).join('/'),
+            guid,
+            id: guid,
+            content: {
+                html: description,
+                text: item.abstract,
+            },
+            image,
+            banner: image,
+        };
+    });
+
+    items = await Promise.all(
+        items.map((item) =>
+            cache.tryGet(item.link, async () => {
+                const { data: detailResponse } = await got(item.link);
+
+                const data = detailResponse.post;
+
+                const title = data.title;
+                const image = data.cover_url;
+                const description = art(path.join(__dirname, 'templates/description.art'), {
+                    image: image
+                        ? [
+                              {
+                                  src: image,
+                                  alt: title,
+                              },
+                          ]
+                        : undefined,
+                    intro: data.abstract,
+                    description: data.content,
+                });
+                const guid = `geekpark-${data.id}`;
+
+                item.title = title;
+                item.description = description;
+                item.pubDate = parseDate(data.published_timestamp, 'X');
+                item.link = new URL(`news/${data.id}`, rootUrl).href;
+                item.category = [...new Set([...data.tags, data.column?.title])].filter(Boolean);
+                item.author = data.authors.map((a) => a.realname ?? a.nickname).join('/');
+                item.guid = guid;
+                item.id = guid;
+                item.content = {
+                    html: description,
+                    text: data.content,
+                };
+                item.image = image;
+                item.banner = image;
+                item.updated = parseDate(data.updated_at);
+
+                return item;
+            })
+        )
+    );
+
+    const data = {
+        title: '',
+        description: '',
+        link: currentUrl,
+        item: items,
+        allowEmpty: true,
+        image: '',
+        author: '',
+    };
+
+    if (column) {
+        data.title = `${response.column.title} | 极客公园`;
+        data.description = response.column.description;
+        data.image = response.column.banner_url;
+    } else {
+        const { data: currentResponse } = await got(currentUrl);
+
+        const $ = load(currentResponse);
+
+        data.title = $('title').text();
+        data.description = $('meta[property="og:description"]').prop('content');
+        data.image = `https:${$('meta[name="og:image"]').prop('content')}`;
+        data.author = $('meta[property="og:site_name"]').prop('content');
+    }
+
+    return data;
+};
+
+export const route: Route = {
+    path: '/:column?',
+    name: '极客公园',
```

Comment:
Route name should not be the same as namespace's name.

Reply:
👌